server/graphql/core/src/lib.rs

```diff
@@ -151,3 +151,28 @@ macro_rules! map_filter {
         }
     }};
 }
+
+#[macro_export]
+macro_rules! map_graphql_diesel_enum {
```

Comment:
Sorry... got distracted on a side quest and implemented this nifty lil macro. Will inform team when this PR gets merged